pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -405,6 +405,19 @@ func TestConvertRbacRulesToFilterConfig(t *testing.T) {
 				},
 			},
 		},
+		{
+			ConfigMeta: model.ConfigMeta{Name: "service-role-7"},
+			Spec: &rbacproto.ServiceRole{
+				Rules: []*rbacproto.AccessRule{
+					{
+						Services: []string{"mysql"},
+						Constraints: []*rbacproto.AccessRule_Constraint{
+							{Key: "envoy.filters.network.mysql_proxy[db.table]", Values: []string{"update"}},
+						},
```

Comment:
Currently in our spec, the key of the "properties" must be an Istio attribute. This violates the assumption.